tests/jerry/regression-test-issue-1386.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2016 Hayun Lee (lhy920806@gmail.com)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+for (y=1950; y<2050; y++) {
+  for (m=0; m<12; m++) {
+    for (d=1; d<32; d++) {
+      var date = new Date(y, m, d);
+      assert(!isNaN(date));
```

Comment:
I think we should find a way to test this since the original problem was arised from untested values.

What about reset the counter when d = 1 (before the d loop) ?